doc/source/regression/dwt-idwt.md

```diff
@@ -9,41 +9,18 @@ kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
-mystnb:
-  execution_allow_errors: true
-  execution_show_tb: true
```

Comment:
`execution_allow_errors` is already set globally in conf.py so it does not need to be set here.

`excution_show_tb` seems like something that should be set globally, so if we want it, we should put it in conf.py.

Reply:
Sure, yes, we should check what the behaviour of `execution_show_tb` is, based on the difference in the tracebacks when it is enabled.

Reply:
Can I leave that to you? I really have no idea what the execution_show_tb setting does.

Reply:
Yes, sorry, I meant that – I would handle it, no worries!

Reply:
Yes, sorry, I meant that I'll handle it, no worries!